scripts/generate_env_vars.sh

```diff
@@ -60,7 +60,7 @@ gen_env_file () {
 # ------------------------------------------------------------------
 # Aether
 # ==================================================================
-AETHER_VERSION=0.9.2
+AETHER_VERSION=0.11.0-rc
```

Comment:
I'm confused... wasn't this already done in https://github.com/eHealthAfrica/aether-bootstrap/pull/18/files?

Reply:
The branch is not up to date with `master` :man_facepalming: 

![image](https://user-images.githubusercontent.com/8459537/47446536-baf0ce80-d7bb-11e8-99dc-cfee0be5f1b3.png)

Reply:
I'm confused. Which branch is out of date? @obdulia-losantos Is this change required?

Reply:
The branch `release-0.11` is 3 commits behind `master` as you can see here: 
https://github.com/eHealthAfrica/aether-bootstrap/branches/all

So part of the changes you are making are already in `master`.

Reply:
Ah, I'm used to github telling me things are out of sync in the PR. Thanks.